resources/definitions/rigid3d_base.def.json

```diff
@@ -106,14 +71,17 @@
         "acceleration_enabled": { "value": "False" },
         "jerk_enabled": { "value": "False" },
 
-        "retraction_amount": { "value": 1.50 },
-        "retraction_prime_speed": { "value": 7 },
-        "retraction_count_max": { "value": 5 },
-        "retraction_combing": { "value": "'noskin'" },
-        "travel_avoid_other_parts": { "value": "False" },
-        
-        "support_z_distance": { "value": "layer_height if layer_height >= 0.16 else layer_height * 2" },
-        "support_interface_enable": { "value": true },
+        "retraction_amount": { "value": 1.5 },
+       ,
+        "travel_retract_before_outer_wall": { "value": "True" },
+        "retraction_combing": { "value": "'all'" },
+        "retraction_combing_max_distance": { "value": 30 },
+
+       ,
```

Comment:
```suggestion
```

Reply:
Excuse me for the inconvenience. Just approved the requested changes by the bot and  things mixed up. Corrected the file and it should be good now.

Reply:
@jellespijker This PR has waited a lot. Is there chance of it being it included in the next release?